geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/dunit/CqQueryDUnitTest.java

```diff
@@ -220,17 +220,17 @@ public void run2() throws CacheException {
 
         for (int i = 0; i < regions.length; i++) {
           createRegion(regions[i], factory.createRegionAttributes());
+          if (getRootRegion("root").getSubregion(regions[i]).isEmpty()) {
```

Comment:
createRegion returns the created region. So instead of looking it up in this way you could just have:
Region createdRegion = createRegion(...);
if (createdRegion.isEmpty()) ...